src/main/java/by/exadel/internship/exception_handing/IncorrectData.java

```diff
@@ -16,6 +16,7 @@
     private UUID errorCode;
 
     public IncorrectData() {
-        this.errorCode = UUID.randomUUID();;
+        this.errorCode = UUID.randomUUID();
+        ;
```

Comment:
What is that?

Reply:
it s for generation exception code error for searching this one in logfile

Reply:
I'm asking about line 20 =)

Reply:
yaaa,  thanks)) i removed it.